docs/detections/alerts-ui-manage.asciidoc

```diff
@@ -109,6 +109,8 @@ The alert details flyout also lists the number and names of cases to which the a
 
 The *Highlighted Fields* section displays the most relevant fields for the alert type. Use this section to inform your triage efforts as you investigate the alert.
 
+NOTE: The *Session ID* field provides a unique ID for the Linux session. To collect it and other session data, you must enable the *Include session data* setting on your Endpoint Security integration policy. Refer to <<enable-session-view, Enable Session View data>> for more information.
```

Comment:
"the Linux session" is a little awkward since otherwise this doesn't seem to be talking about a specific session. Alternatively, something like "the Linux session that generated the alert" but that's a bit wordy.

Also li'l shared attribute fix:
```suggestion
NOTE: The *Session ID* field provides a unique ID for Linux sessions. To collect it and other session data, you must enable the *Include session data* setting on your {endpoint-cloud-sec} integration policy. Refer to <<enable-session-view, Enable Session View data>> for more information.
```